### PR TITLE
Running tests with py.test

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-6842

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: tox.ini [hunk 1]
```diff
@@ -12,8 +12,7 @@ deps =
     -rtest_requirements.txt
 
 commands =
-    django-admin.py test auth_backends --settings=test_settings --with-coverage --cover-package=auth_backends
-    coverage report
+    py.test {posargs}
```
Comment (rlucioni) [inline tox.ini]: Just call `pytest`. `py.test` is the old way; the [docs](http://doc.pytest.org/en/latest/) recommend `pytest`.

File: pytest.ini [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+DJANGO_SETTINGS_MODULE=test_settings
+addopts = --cov auth_backends --cov-report term-missing --cov-report xml
```
Comment (rlucioni) [inline pytest.ini]: Consider adding a `testpaths` directive to speed up collection.

Comment (clintonb) [inline pytest.ini]: Done.

**Pull request merged**